src/cairoWriter/writers/cairoContractWriter.ts

```diff
@@ -105,30 +106,51 @@ export class CairoContractWriter extends CairoASTNodeWriter {
             '    DUMP_WARP_STORAGE_ITER(index, ptr);',
             '    return ();',
             '}',
-            '@external',
-            'func DUMP_WARP_STORAGE{syscall_ptr : felt*, pedersen_ptr : HashBuiltin*, range_check_ptr}(length : felt) -> (data_len : felt, data: felt*){',
+            '#[external]',
```

Comment:
This function is not fully converted, also it is fixed by the storage mappings branch. Please revert to it's original state

Reply:
The reason why is not fully converted is because it has not strightforward convertion. It should be handled as you said as part of the storage implementation. It have that old and invalid syntax just to mark that is an unadressed problem in this PR.